src/parser/flow_lexer.ml

```diff
@@ -58,6 +58,14 @@ let scinumber = [%sedlex.regexp?
 let wholenumber = [%sedlex.regexp? underscored_digit, Opt '.']
 let floatnumber = [%sedlex.regexp? Opt underscored_digit, '.', underscored_decimal]
 
+let binbigint = [%sedlex.regexp? binnumber, 'n']
+let octbigint = [%sedlex.regexp? octnumber, 'n']
+let legacyoctbigint = [%sedlex.regexp? legacyoctnumber, 'n']
+let hexbigint = [%sedlex.regexp? hexnumber, 'n']
+let scibigint = [%sedlex.regexp? decintlit, ('e' | 'E'), Opt ('-' | '+'), underscored_digit, 'n']
+let wholebigint = [%sedlex.regexp? underscored_digit, 'n']
```

Comment:
Make sure to add a test for something like `1.n`. It doesn't look supported by the bigint proposal but `floatbigint` won't allow it.

Reply:
What about `1..n`?